jenkins_branches.sh

```diff
@@ -8,11 +8,20 @@ VENV_PATH="${HOME}/venv/${JOB_NAME}"
 
 pip install -q ghtools
 
-REPO="alphagov/metadata-api"
+REPO=alphagov/metadata-api
+export GOPATH=$PWD/gopath
+GO_GITHUB_PATH=$GOPATH/src/github.com
+BUILD_PATH=$GO_GITHUB_PATH/$REPO
+
+rm -rf $GOPATH && mkdir -p $GOPATH/bin $BUILD_PATH
+
+rsync -a ./ $BUILD_PATH --exclude=gopath
+
 gh-status "$REPO" "$GIT_COMMIT" pending -d "\"Build #${BUILD_NUMBER} is running on Jenkins\"" -u "$BUILD_URL" >/dev/null
 
-if make; then
+if cd $BUILD_PATH && make; then
```

Comment:
Now that we have a `jenkins.sh` it would be better for this script to call it instead of having to duplicate the build logic. This is the pattern used in several other repos on GOV.UK (eg https://github.com/alphagov/frontend/blob/master/jenkins_branches.sh)